OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoStatsLogic.cs

```diff
@@ -79,8 +79,10 @@ public GameInfoStatsLogic(Widget widget, ModData modData, World world, OrderMana
 					checkbox.GetText = () => mo.Objectives.First().Description;
 				}
 
+				var failed = modData.Translation.GetString(Failed);
+				var inProgress = modData.Translation.GetString(InProgress);
 				statusLabel.GetText = () => player.WinState == WinState.Won ? Accomplished :
```

Comment:
You missed `Accomplished`.

Reply:
This makes me think the approach we've taken is way too fragile. Turning the current simple string fields that just hold the id of the translatable string into something more complicated like a struct that holds the key and also does the call to `Translation.GetString()` internally might give us something much more robust. Also kinda related is the recent #20247

Reply:
we could incorporate `modData.Translation.GetString` into `Label.GetText`. But for that we first need to extract yaml strings to fluent